docs/next/guides/getting-started/setting-options.md

```diff
@@ -14,319 +14,219 @@ tags:
 ```
-  
 
-### Columns
+#### `cells`
 
-Configuration options that are provided using second-level object.
+The [cells](@/api/metaSchema.md#cells-2) option works slightly differently. The `cells` option is a function that invokes the logic in it before the table's rendering cycle. The options returned by that function always overwrite the entire table or column options. Thus, if you implement logic incorrectly, you won't be able to change cells' state from API or UI - as the values will be constantly overwritten by the `cells` function. You can think of this option as an opportunity to transfer the responsibility of keeping the table cells option to your application.
 
-```js
-const container = document.querySelector('#example');
+If you want to stop, overwrite the cell meta object for particular cells, just return an empty object.
 
-new Handsontable(container, {
-  columns: {
-    option: 'value'
+```js
+// ...
+cells(row, column) {
+  if (row === 1 && column === 1) {
+    return { readOnly: true };
   }
-});
+},
+// ...
+
+hot.getCellMeta(0, 0).readOnly === false; // By default the option is "false"
+hot.getCellMeta(1, 1).readOnly === true; // the "cells" overwrites the value
+
+// In this case, an attempt to change the state will have no effect. The `readOnly`
+// property always returns `true`.
+hot.setCellMeta(1, 1, 'readOnly', false);
 ```
 
-### Cells
+## Show time
+
+With a combination of the above configuration layers, you're able to manage a more complicated setup.
 
-Configuration options that are provided using second-level function.
+Consider the following example:
 
+::: example #example5 --html 1 --js 2
+```html
+<div id="example5" class="hot"></div>
+```
 ```js
-const container = document.querySelector('#example');
+const container = document.querySelector('#example5');
 
-new Handsontable(container, {
-  cells(row, col, prop) {
-  }
+const hot = new Handsontable(container, {
+  data: Handsontable.helper.createSpreadsheetData(5, 10),
+  readOnly: true,
+  columns: [
+    { readOnly: false },
+    {},
+    {},
+    {},
+    {},
+    {},
+    {},
+    {},
+    {},
+    {},
+  ],
+  cells(row, column) {
+    if (row === 0 && column === 0) {
+      return { readOnly: true }
+    }
+
+    if (row === 2 && column === 2) {
+      return { readOnly: false }
+    }
+  },
+  width: 'auto',
+  height: 'auto',
+  rowHeaders: true,
+  colHeaders: true,
+  licenseKey: 'non-commercial-and-evaluation'
 });
 ```
+:::
+
+The above notation results in all cells being read-only, except for the first column, which is editable. The top-left cell and cell at row and column index `3, 3` are still read-only. Their properties are individually managed within the [cells](@/api/metaSchema.md#cells-2) option.
 
 ## All options
 
 The most up to date list of options is available in the [Options](@/api/metaSchema.md) section of the API Reference. The table below is just a cheatsheet that can be useful if you're already familiar with the Handsontable API.
```

Comment:
Please remove " The table below is just a cheatsheet that can be useful if you're already familiar with the Handsontable API."